### DISCUSSION TITLE
Problem accessing xaml code of Uno NuGet package

### DISCUSSION THREAD

**Discussion by HavenDV** [unoplatform/uno#9456] [Q&A] [answered]
I have published the Syncfusion Uno Chart library as a NuGet package for use in my projects. But I'm having trouble accessing it.
Everything worked correctly when I used the library locally.
https://github.com/HavenDV/Uno.SfChart
<img width="426" alt="image" src="https://user-images.githubusercontent.com/3002068/183826999-5cd18a71-9de5-402f-ae9f-91135b204bdd.png">

**Comment by jeromelaban** [comment, marked as answer]
Are you able to use the dictionary without specifying `SyncFunction.SfChart.Uno` in the path? Can you also make sure that the file is having `Page` as the build action?

**Reply by HavenDV** [reply]
Thank you. The problem turned out to be that I renamed the project, forgetting to rename all internal ms-appx links to the new name:
https://havendv.github.io/Uno.SfChart/
